container/common/helpers.go

```diff
@@ -258,6 +258,13 @@ func assignDeviceNamesToPerDiskStats(namer DeviceNamer, diskStats ...[]info.PerD
 	}
 }
 
+// FilterCgroupPaths delete cgroup paths which we do't need base on includeMetrics
+func FilterCgroupPaths(cgroups map[string]string, includedMetrics container.MetricSet) {
+	if !includedMetrics.Has(container.DiskIOMetrics) {
```

Comment:
we don't currently do this for other ignored metrics.  What is the benefit of removing the blkio cgroup from the list of cgroups?

Reply:
As far as I can see, if we don't remove the blkio cgroup, it would keep extracting data from /sys/fs/cgroup/blkio/(ref:https://github.com/google/cadvisor/blob/45d3b774d690cf0221f06fc799e62e5cd8715e58/vendor/github.com/opencontainers/runc/libcontainer/cgroups/fs/blkio.go#L115). And the data it extracted would be just dropped cause we have disabled the diskIO metrics. So here I am trying to do is avoiding useless IO request when diskIO was disabled.